app/views/partials/multipleCheckbox.scala.html

```diff
@@ -0,0 +1,49 @@
+@import views.html.partials.inputSingleCheckbox
+@import controllers.util.InputNameAndValue
+@import views.html.partials.errorMessage
+@(fieldId: String,
+fieldName: String,
+fieldDescription: String,
+multiValue: Boolean=false,
+options: Seq[InputNameAndValue],
+selectedOptions: Seq[InputNameAndValue],
+fieldErrors: List[String] = Nil
+)(implicit messages: Messages)
+
+<div class="govuk-form-group @{if(fieldErrors.nonEmpty) "govuk-form-group--error" else ""}">
+    <fieldset class="govuk-fieldset" aria-describedby="foi-hint">
+        <legend class="govuk-fieldset__legend govuk-fieldset__legend--m">
+            <h2 class="govuk-fieldset__heading">
+                @fieldName
+            </h2>
+        </legend>
+        <div class="govuk-hint">
+        @fieldDescription
+        </div>
+
+        @errorMessage(fieldId, fieldErrors)
+        <div class="govuk-checkboxes" data-module="govuk-checkboxes" id="@fieldId">
+            <div class="govuk-grid-row">
+        @options.zipWithIndex.map {
+            case (foiCode, index) => {
+                @if(index == 0 || index % 15 == 0) {<div class="govuk-grid-column-one-third">}
```

Comment:
What is this doing?

Reply:
In the prototype there are 3 columns for the check boxes, each column consisting of 15 check boxes. This just ensures there's a new column created for every 15 check boxes.